### PR TITLE
[PSL-147] store fingerprints and rq symbols

### PR DESCRIPTION
**Pull request by mateeullahmalik**

### CODE REVIEW

**Review by a-ok123 — APPROVED** [review]

**Comment by a-ok123** [comment]
LGTM

File: supernode/services/artworkregister/task.go [hunk 1]
```diff
@@ -796,6 +823,28 @@ func (task *Task) storeThumbnails(ctx context.Context) error {
...
+		}
+
+		return nil
+	}
+
+	if err := store(ctx, task.Ticket.AppTicketData.DDAndFingerprintsIDs); err != nil {
+		return fmt.Errorf("store ddAndFp files: %w", err)
+	}
+
+	if err := store(ctx, task.Ticket.AppTicketData.RQIDs); err != nil {
```
Comment (nguyen202) [inline supernode/services/artworkregister/task.go:841]: any wrong here?
I don't think we store RQIDs into kademlia.
RQIDs here are hashs of RQIDs file - actualy is key to refer to p2p data of `SHA3-256(compress(Base64(rq_ids).Base64(signature).<cnt>))`.
I think you should store : `compress(Base64(rq_ids).Base64(signature).<cnt>))`

File: supernode/services/artworkregister/task.go [hunk 2]
```diff
@@ -796,6 +823,28 @@ func (task *Task) storeThumbnails(ctx context.Context) error {
...
+		for _, idFile := range ids {
+			if _, err := task.p2pClient.Store(ctx, []byte(idFile)); err != nil {
+				return err
+			}
+		}
+
+		return nil
+	}
+
+	if err := store(ctx, task.Ticket.AppTicketData.DDAndFingerprintsIDs); err != nil {
```
Comment (nguyen202) [inline supernode/services/artworkregister/task.go:837]: I don't think you should store DDAndFingerprintsIDs into p2p, should store : 
`compressed(Base64(dd_and_fingerprints).Base64(signatureSN1).Base64(signatureSN2).Base64(signatureSN3).1234)`

File: supernode/services/artworkregister/task.go (hunk 1 above)
Comment (mateeullahmalik) [reply supernode/services/artworkregister/task.go:841]: correct, Good eyes! @nguyen202

**Pull request merged**